db/schema.rb

```diff
@@ -124,16 +113,19 @@
     t.string   "city"
     t.string   "state"
     t.string   "zip_code"
-    t.datetime "created_at",               null: false
-    t.datetime "updated_at",               null: false
+    t.datetime "created_at",                               null: false
+    t.datetime "updated_at",                               null: false
     t.integer  "age"
     t.string   "gender"
     t.string   "race"
     t.string   "size"
     t.string   "wish_list"
     t.integer  "recipient_family_id"
+    t.integer  "membership_id"
```

Comment:
Could also be `t.references :membership, foreign_key: true` maybe? Same for the recipient family, organization, and user. That should automatically create the correct indices in the schema.rb instead of hand creating them